src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Add the eslint plugin and enforce the basic react rules. for example, you have to use double quotes instead of single.